homeassistant/components/webostv/strings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "config": {
+    "flow_title": "LG webOS Smart TV ({name})",
```

Comment:
The name of the integration will already show in the UI. You can drop that and just use {name}